social-messenger/src/App.js

```diff
@@ -80,6 +80,20 @@ const App = () => {
     }
   }, [isMobileNavVisible]);
 
+  useEffect(() => {
+    /*
+     * Get the actual rendered window height to set the container size properly
+     * as in some browsers (like iOS) the browser nav bar might override
+     * the app.
+     */
+    const appHeight = () => {
+      const doc = document.documentElement;
+      doc.style.setProperty('--app-height', `${window.innerHeight}px`);
+    };
+    window.addEventListener('resize', appHeight);
```

Comment:
It would be nice to remove the listener on component unmount

Reply:
Hecho! https://github.com/GetStream/website-react-examples/pull/23/commits/2816313c4e9702fad2d44e26c153d5c80b86468d